ddtrace/internal/_encoding.pyx

```diff
@@ -589,6 +585,12 @@ cdef class MsgpackEncoderV03(MsgpackEncoderBase):
             ret = pack_number(&self.pk, span.duration_ns)
             if ret != 0: return ret
 
+            if has_error:
+                ret = pack_bytes(&self.pk, <char *> b"error", 5)
+                if ret != 0: return ret
+                ret = msgpack_pack_long(&self.pk, <long> (1 if span.error else 0))
```

Comment:
```suggestion
                ret = msgpack_pack_long(&self.pk, <long> 1)
```

?

since we checked `has_error`, `span.error` here can only be 1/True

Reply:
I guess _technically_ error could be not 1 or 0 in which case this is still doing something... but I think our unofficial contract is such that it can only be 1 or 0 so I'm ok with making this change.